FWCore/Framework/src/global/EDFilterBase.cc

```diff
@@ -110,8 +110,8 @@ namespace edm {
       r.setProducer(this);
       Run const& cnstR = r;
       const EventSetup c{ci, static_cast<unsigned int>(Transition::EndRun), esGetTokenIndices(Transition::EndRun)};
-      this->doEndRunProduce_(r, c);
       this->doEndRunSummary_(r, c);
+      this->doEndRunProduce_(r, c);
```

Comment:
@Dr15Jones Do we have an existing test for EDFilters? If not, should we consider extending the `{global,stream,limited}_module_t.cppunit.cc` to cover EDFilters as well?

Reply:
It would be good to extend the test for EDFilters as well.

Reply:
I added separate files to test stream/limited/global EDFilters as well (by copy-paste+replace, I tried to play template tricks to avoid copy-paste but after some time I essentially ended up with circular dependencies).